config/reader.param.yaml

```diff
@@ -0,0 +1,6 @@
+/**:
+  ros__parameters:
+    hardware_id: 3
+    circuit_id: 0
+    bit_rate: 500000
+    enable_echo: false
```

Comment:
Let me confirm this again, https://github.com/tier4/lexus_description.iv.universe/pull/11#discussion_r581021300.

It seems you reverted the name of pacmod3 file, but kvaser files are not changed.
https://github.com/tier4/lexus_description.iv.universe/pull/11/files/1e28427e51e51385fcac90a35dcfb8030a394366..c3ba176b7595d8583058668e60925cbe615deb95

I think it might be acceptable because these files are newly created, but a bit strange because they already have `bridge_params.yaml` and these don't have the same naming convention.
https://github.com/astuff/kvaser_interface/blob/dashing-devel/launch/bridge_params.yaml

I'd like to hear your idea!

Reply:
Thank you for your comment! I agree your suggestion. I will rename param file to unify naming convention.